tests/checkup_test.go

```diff
@@ -331,6 +331,7 @@ func newConfigMap() *corev1.ConfigMap {
 			"spec.param.vmContainerDiskImage":             vmContainerDiskImage,
 			"spec.param.testDuration":                     "1m",
 			"spec.param.verbose":                          "true",
+			"spec.param.trafficGeneratorPacketsPerSecond": "8m",
```

Comment:
Could you please put the new config bellow `spec.param.trafficGeneratorImage`?
This way, all of the traffic gen config would be closer together.

Reply:
DONE